app/templates/public/index.hbs

```diff
@@ -1,3 +1,10 @@
-<div class="event description">
-  {{sanitize model.event.description}}
-</div>
\ No newline at end of file
+<div class='event'>
```

Comment:
Since this class was changed, you'd have to update it at `app/styles/pages/public-event.scss#L87`